src/lib/selections.test.ts

```diff
@@ -0,0 +1,1232 @@
+import { expect } from 'vitest'
```

Comment:
This files tests were here to capture previous behaviour to give me some piece of mind when refactoring